FileGetContentsLoader.php

```diff
@@ -31,8 +31,12 @@ public function loadDocument($url)
 
             $streamContextOptions = array(
               'method'  => 'GET',
-              'header'  => "Accept: application/ld+json, application/json; q=0.9, */*; q=0.1\r\n",
-              'timeout' => Processor::REMOTE_TIMEOUT
+              'header'  => "Accept: application/ld+json, application/json; q=0.9, */*; q=0.1\r\n".
+                           // Fix F. Michel 2017-12-06 - Some Web API require a User-Agent.
+                           // E.g. MusicBrainz returns error 403 if there is none.
+                           "User-Agent: JsonLD.php\r\n",
```

Comment:
Please change the user agent name, remove the comments and move the dot on to the next line

```
              'header'  => "Accept: application/ld+json, application/json; q=0.9, */*; q=0.1\r\n"
                             . "User-Agent: JsonLD\r\n",
```